parm/metplus/separate_fcst_obs_info.py

```diff
@@ -0,0 +1,249 @@
+    fgs_fields_levels_threshes_fcst = {}
+    fgs_fields_levels_threshes_obs = {}
+    for field_group, fields_levels_threshes_both in fgs_fields_levels_threshes_both.items():
+
+        # Loop over the field names associated with the current field group.
+        #fields_levels_threshes_fcst = {}
+        #fields_levels_threshes_obs = {}
+        fields_levels_threshes_fcst = []
+        fields_levels_threshes_obs = []
+        for field_both, levels_threshes_both in fields_levels_threshes_both.items():
+    
+            # Parse the current field name to extract the forecast field name and the
+            # observation field name.
+            tmp = field_both.split('|')
+            num_sep_chars = len(tmp) - 1
+            field_fcst = tmp[0].strip()
+            if num_sep_chars == 0:
+                field_obs = field_fcst
+            elif num_sep_chars == 1:
+                field_obs = tmp[1].strip()
+            else:
+                error_msg
+    
+            print(f'        field_fcst = {field_fcst}')
+            print(f'        field_obs = {field_obs}')
+    
+            # Loop over the levels associated with the current field.
+            levels_threshes_fcst = {}
+            levels_threshes_obs = {}
+            for level_both, threshes_both in levels_threshes_both.items():
+    
+                # Parse the current level to extract the forecast level and the observation
+                # level.
+                tmp = level_both.split('|')
+                num_sep_chars = len(tmp) - 1
+                level_fcst = tmp[0].strip()
+                if num_sep_chars == 0:
+                    level_obs = level_fcst
+                elif num_sep_chars == 1:
+                    level_obs = tmp[1].strip()
+                else:
+                    error_msg
+    
+                # Loop over the thresholds associated with the current level.
+                threshes_fcst = []
+                threshes_obs = []
+                for thresh_both in threshes_both:
+    
+                    # Parse the current threshold to extract the forecast threshold and the
+                    # observation threshold.
+                    tmp = thresh_both.split('|')
+                    num_sep_chars = len(tmp) - 1
+                    thresh_fcst = tmp[0].strip()
+                    if num_sep_chars == 0:
+                        thresh_obs = thresh_fcst
+                    elif num_sep_chars == 1:
+                        thresh_obs = tmp[1].strip()
+                    else:
+                        error_msg
+    
+                    # Append the current forecast threshold to the list of forecast thresholds
+                    # corresponding to the current field and level.  Then do the same for the
+                    # observations.
+                    threshes_fcst.append(thresh_fcst)
+                    threshes_obs.append(thresh_obs)
+    
+                print(f'            threshes_fcst = {threshes_fcst}')
```

Comment:
Can all these prints be tied to a `verbose` or similar flag? This is a lot of printing going to log files by default.